docs/user-docs/external-process-waiting.md

```
@@ -0,0 +1,114 @@
+# When the run completes, the test fails with exit 1 (done).
+checkCommand: 'test "$(gh run view 12345 --json status -q .status)" != "completed"'
+
+# Phase 2: check the conclusion field for success.
+successCheck: 'test "$(gh run view 12345 --json conclusion -q .conclusion)" = "success"'
+```
+
+The `test != "completed"` pattern matches the exit code convention: exit 0 while the condition holds (still running), exit 1 when it no longer holds (done).
+
+## Lifecycle
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)